### PR TITLE
Add documentation for other debug uORB topics

### PR DESCRIPTION
**Pull request by jlecoeur**
Associated PX4 PR: https://github.com/PX4/Firmware/pull/7373

### CODE REVIEW

File: en/middleware/mavlink.md [hunk 1]
```diff
@@ -167,6 +167,18 @@ MavlinkReceiver::handle_message(mavlink_message_t *msg)
...
+## Alternative to Creating Custom MAVLink Messages
+Sometimes there is the need for custom MAVLink message whose content is not fully defined.
+
+For example when using MAVLink to interface PX4 with an embedded device, the messages that are exchanged between the autopilot and the device may go through several iterations before they are stabilized.
+In this case, it can be time-consumming and error-prone to regenerate the MAVLink headers, and make sure both devices use the same version of the protocol.
+
+An alternative - and temporary - solution is to repurpose [debug messages](../debug/debug_values.md). 
+Instead of creating a custom MAVLink message `CA_TRAJECTORY`, you can send a message `DEBUG_VECT` with the string key `CA_TRAJ` and data in the `x`, `y` and `z` fields.
+
+** Note: This solution is not efficient as it sends character string over the network and involves comparison of strings. It should be used for development only! **
```
Comment (hamishwillee) [inline en/middleware/mavlink.md]: Format for notes should be as shown below (renders the content in a box in gitbook, looks OK in github.

```
> **Note** This solution is not efficient as it sends character string over the network and involves comparison of strings. It should be used for development only! 
```

Comment (jlecoeur) [reply en/middleware/mavlink.md]: Thanks @hamishwillee I fixed it

File: en/middleware/mavlink.md [hunk 2]
```diff
@@ -167,6 +167,18 @@ MavlinkReceiver::handle_message(mavlink_message_t *msg)
 		...
  	}
 ```
+
+## Alternative to Creating Custom MAVLink Messages
+Sometimes there is the need for custom MAVLink message whose content is not fully defined.
```
Comment (hamishwillee) [inline en/middleware/mavlink.md]: Should be "a" custom ...". 
```
Sometimes there is the need for a custom MAVLink message with content that is not fully defined.
```

File: en/middleware/mavlink.md [hunk 3]
```diff
@@ -167,6 +167,18 @@ MavlinkReceiver::handle_message(mavlink_message_t *msg)
...
+
+For example when using MAVLink to interface PX4 with an embedded device, the messages that are exchanged between the autopilot and the device may go through several iterations before they are stabilized.
+In this case, it can be time-consumming and error-prone to regenerate the MAVLink headers, and make sure both devices use the same version of the protocol.
+
+An alternative - and temporary - solution is to repurpose [debug messages](../debug/debug_values.md). 
+Instead of creating a custom MAVLink message `CA_TRAJECTORY`, you can send a message `DEBUG_VECT` with the string key `CA_TRAJ` and data in the `x`, `y` and `z` fields.
```
Comment (hamishwillee) [inline en/middleware/mavlink.md]: Might be cool to add an example code fragment - or link to where this is done.

File: en/debug/debug_values.md [hunk 4]
```diff
@@ -1,9 +1,23 @@
...
+# Send and Receive Debug Values
 
 It is often necessary during software development to output individual important numbers.
-This is where the generic `NAMED_VALUE` packets of MAVLink come in.
+This is where the generic `NAMED_VALUE_FLOAT`, `DEBUG` and `DEBUG_VECT` packets of MAVLink come in.
 
-## Files
+## Mapping between MAVLink Debug Messages and uORB Topics
+
+MAVLink debug messages are translated to/from uORB topic.
```
Comment (hamishwillee) [inline en/debug/debug_values.md]: topics (not topic)

**Comment by hamishwillee** [comment]
@jlecoeur I've added some comments, and other than those this is clear to me. However I don't know technically whether it is correct - @bkueng can you review for technical accuracy?

**Comment by bkueng** [comment]
This looks correct.

**Comment by jlecoeur** [comment]
Thank you @hamishwillee. I corrected the grammar and added an example code.

**Review by TSC21 — APPROVED** [review]
lgtm both grammatically and technically

**Comment by hamishwillee** [comment]
THanks @jlecoeur - merged!

**Pull request merged**